CodeEdit/NavigatorSidebar/SourceControlNavigator/Views/Changes/ChangesView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ChangesView.swift
+//  CodeEdit
+//
+//  Created by Nanashi Li on 2022/05/05.
+//
+
+import SwiftUI
+import CodeEditUI
+import Git
+
+struct ChangesView: View {
+
+    @ObservedObject
+    var model: SourceControlModel
+
+    @State
+    var selectedFile: ChangedFiles.ID?
+
+    /// Initialize with GitClient
+    /// - Parameter gitClient: a GitClient
+    init(workspaceURL: URL) {
+        self.model = .init(workspaceURL: workspaceURL)
```

Comment:
instead of passing the workspaceURL we can pass the `SourceControlModel` directly

Reply:
It's used for ChangedFileItemView to open files in the finder also

Reply:
could you indicate me in which file this is used please? It's a weird pattern

Reply:
seems that `ChangedFileItemView` has the workspaceURL too so it should be fine if we remove it from here

Reply:
If you read my comment right you would see I said it is used to open the finder at the file directory.

Reply:
@nanashili I definitely read your comment right, but a link to the actual reference might be helpful for the context. Thank you!